README.md

```diff
@@ -1,23 +1,22 @@
-# zfs-vm-storage
+# zfs-storage
 
 This role sets up ZFS filesystem that can be shared via NFS.
 
-
 ## Requirements
 
-an apt-based package manager with source version 16.04 or later, as zfs isn't included in older versions.
-
+An apt-based package manager with source version 16.04 or later, as zfs isn't included in older versions.
 
 ## Role Variables
 
-| Name                  | Description                                                                                 |
-|-----------------------|---------------------------------------------------------------------------------------------|
-| `parent_fs`           | existing parent zfs filesystem for all filesystems and zvols (Default: `tank`)              |
-| `zfs_storage_defaults`| dict containing zfs attributes that will be applied to all configured zfs filesystems/zvols |
-| `filesystems`         | list of filesystems defined by a `name` and a dict of `attributes` (both mandatory)         |
-| `zvols`               | list of zvols defined by a `name` and a dict of `attributes` (both mandatory)               |
+| Name                  | Mandatory | Default | Description                                                                                 |
```

Comment:
I do in my text editor. The text editor where I just have to type `:Table<Tab>` to indent the table. Just do it!